api/nivs.js

```diff
@@ -0,0 +1,126 @@
+import { backendsPromise, visualisationApiUrl } from '~/api/backends/'
+import axios from 'axios'
+import { instanceId } from '~/constants/localUUIDs'
+
+const stateFileName = 'state.json'
+const builderId = 'a734e3e7-ca10-41f2-9638-a19710d6430d'
```

Comment:
With the current, you'd have to retrieve this from the API I think because the ID is different in each NIVS env.

I could change it so they're hardcoded though if that's going to be a pain.

Reply:
Ah thats good to know, I pretty much left this alone beyond minor refactoring because the uploading of Templates is disabled for now. Thought it'd be easier to fix it once templates was properly in place.

Reply:
Yeah, makes sense, can talk about it when we get there...